src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -3490,12 +3488,6 @@ void GetInputZoneEvaporativeCoolerUnit(EnergyPlusData &state)
             // with used for ZONECOOLINGLOADVARIABLESPEEDFAN Cooler Unit Control Method
             thisZoneEvapUnit.fanOp = HVAC::FanOp::Cycling;
 
-            FanVolFlow = 0.0;
-            if (errFlag) {
-                ShowContinueError(state, format("specified in {} = {}", CurrentModuleObject, thisZoneEvapUnit.Name));
-                ErrorsFound = true;
-            }
-
```

Comment:
[src/EnergyPlus/EvaporativeCoolers.cc:3494]:(style),[knownConditionTrueFalse],Condition 'errFlag' is always false
[src/EnergyPlus/EvaporativeCoolers.cc:1187]:(style),[variableScope],The scope of the variable 'SizingDesRunThisZone' can be reduced.